main.go

```diff
@@ -670,7 +676,20 @@ func main() {
 						metricsv1.WithReadMiddleware(metricsv1.WithEnforceTenancyOnMatchers(cfg.metrics.tenantLabel)),
 						metricsv1.WithReadMiddleware(metricsv1.WithEnforceAuthorizationLabels()),
 						metricsv1.WithUIMiddleware(authorization.WithAuthorizers(authorizers, rbac.Read, "metrics")),
-					))
+						metricsv1.WithAlertmanagerAlertsReadMiddleware(
```

Comment:
Nice! Let's also refactor the other middleware functions to be variadic then, for consistency? 🙂

Reply:
yes will do, but I think it makes morse sense in a follow up PR?

Reply:
Sure!